docs/docs-content/automation/automation.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: "Automation"
+title: "Automation"
+description: "Learn how to use automation tools with Palette and Palette VerteX."
+hide_table_of_contents: false
+sidebar_position: 0
+sidebar_custom_props:
+  icon: "terminal"
+tags: ["automation"]
+---
+
+This section contains documentation and guides essential for automating tasks with Palette. These resources cover the
+Palette CLI, Terraform provider, and Crossplane provider.
```

Comment:
```suggestion
Palette CLI, Palette Terraform provider, and Palette Crossplane provider.
```
I think we need to prefix with Palette, otherwise it's not clear what provider for Crossplane and Terraform we are referring to.

Reply:
:white_check_mark: